typescript-operators-and-expressions/main.ts

```diff
@@ -33,6 +33,7 @@ const isSparta: boolean = headCount === 300;
 console.log('the value of isSparta', isSparta);
 console.log('the typeof isSparta', typeof isSparta);
 
-const motto: string = fullName + '' + ' is the GOAT';
+const motto: string = fullName 
+const motto: string += 'is the GOAT';  
```

Comment:
Not quite! If you are going to reassign a variable, you can't use `const` right?